languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml

```diff
@@ -420,7 +420,7 @@ title="Easy editing stylesheet" ?>
              <token postag="C.+|I.+|enfim|opa" postag_regexp="yes"/>
```

Comment:
again: `<token postag="C.+|I.+|enfim|opa" postag_regexp="yes"/>` does not seem to work
`C.+|I.+` seem to be postags and `enfim|opa` seem to be actual words.
they need to be  separated into two APs